easybuild/easyblocks/generic/intelbase.py

```diff
@@ -203,11 +204,17 @@ def setup_local_home_subdir(self):
         except OSError, err:
             raise EasyBuildError("Failed to symlink %s to %s: %s", self.home_subdir_local, self.home_subdir, err)
 
-    def prepare_step(self):
+    def prepare_step(self, requires_runtime_license=True):
         """Custom prepare step for IntelBase. Set up the license"""
         super(IntelBase, self).prepare_step()
 
-        if self.cfg['licensed']:
+        # Decide if we need a license or not
+        if not self.cfg['requires_runtime_license'] or not requires_runtime_license:
+            self.requires_runtime_license = False
+        else:
+            self.requires_runtime_license = True
```

Comment:
Isn't this always `True` regardless of the easyconfig setting since the default value of `requires_runtime_license` is `True`